src/utils/withWidth.js

```diff
@@ -0,0 +1,120 @@
+// @flow weak
+
+import React, { Component, PropTypes } from 'react';
+import EventListener from 'react-event-listener';
+import createHelper from 'recompose/createHelper';
+import createEagerFactory from 'recompose/createEagerFactory';
+
+const BREAKPOINTS = [
```

Comment:
@oliviertassinari Can we export a constant from this file?: https://github.com/oliviertassinari/material-ui/blob/6f7e30faef707607a9582a2cca0fc7c2e3d18793/src/styles/breakpoints.js

Then we can grab the keys from that object.